cmd/server/main.go

```diff
@@ -124,44 +126,44 @@ func realMain(ctx context.Context) error {
 	// Install the CSRF protection middleware.
 	csrfController := csrfctl.New(ctx, h)
 	// TODO(mikehelmick) - there are many more configuration options for CSRF protection.
-	r.Use(csrf.Protect(config.CSRFAuthKey,
+	r.Use(middleware.ConfigureCSRF(ctx, config.CSRFAuthKey,
```

Comment:
If we're using our own middleware wrapper here, why not set defaults so you only have to pass in config and csrfController?

Reply:
Done